static/templates/left_sidebar.hbs

```diff
@@ -48,6 +48,16 @@
                 </a>
                 <span class="arrow starred-messages-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
+            <li class="top_left_drafts top_left_row hidden-for-spectators">
+                <a href="#drafts">
+                    <span class="filter-icon">
+                        <i class="fa fa-pencil" aria-hidden="true"></i>
+                    </span>
+                    {#- squash whitespace -#}
```

Comment:
This should be `{{~!-- squash whitespace --~}}` -- the syntax changed when switching files.

BTW I'd recommend always testing changes after any rebase where you have to do anything; it's sadly more likely than not that something breaks, and the manual QA is often a nice opportunity to refresh whether one feels good about the feature.

Reply:
I took care of switching this.